docs2/site/docs/migrations/migration7.md

```diff
@@ -598,3 +598,10 @@ require them to resolve to GraphQL Object types. GraphQL.NET 7.2+ now enforces t
 Due to the above validation, resolvers defined on fields of interface types are never be used by
 GraphQL.NET. As such, resolvers are not generated for fields of `AutoRegisteringInterfaceGraphType`s
 in version 7.2+.
+
+### 18. `SchemaValidationVisitor` has more runtime checks
+
+From version 7.4 on, only root fields of subscriptions should set `StreamResolver` and only object
+output types should set `Resolver` properties. This change may be potentially breaking for your apps,
+because schema will throw exception during initialization. To get rid of that exception just stop
+setting these properties for fields mentioned in exception message.
```

Comment:
```suggestion
Starting with version 7.4, the `StreamResolver` property should be set exclusively for root fields
in subscriptions, while the `Resolver` property should only be assigned to object output types'
fields. Fields within union types, interface types and input object types should not set these
properties. This modification may lead to potential disruptions in your applications, as the
schema may trigger an exception during initialization. To eliminate this exception, simply
cease assigning these properties for the fields specified in the exception message.
```

I suggest adding an explicit notice that union/interface/input objects cannot have resolver set.  Then I asked ChatGPT to rewrite it.

Reply:
> Fields within union types

Strictly speaking unions have no fields. They consist of other object graph types. 

Leave as as?

Reply:
Whatever you think.  I'll approve now and you can take out 'unions' out of there if you like.